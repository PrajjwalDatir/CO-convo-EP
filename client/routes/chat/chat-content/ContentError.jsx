import React from 'react';
import ContentContainer from './ContentContainer';
import channelErrorImage from '#/assets/images/undraw_lost.svg';

function ContentError() {
  return (
    <ContentContainer>
      <div className="flex flex-col items-center justify-center">
        <div className="ml-8">
          <h1
            className="text-5xl whitespace-normal leading-tight font-bold"
            style={{ fontFamily: 'Raleway, sans-serif' }}
          >
            A server error has occurred.
          </h1>
          <p className="text-gray-700 text-xl mt-2 mb-5">
            Please try again later
          </p>
        </div>
        <img
          className="w-3/5"
          alt="Man looking at map"
          src={channelErrorImage}
        />
      </div>
    </ContentContainer>
  );
}

export default ContentError;
