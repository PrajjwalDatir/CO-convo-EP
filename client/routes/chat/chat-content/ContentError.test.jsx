import React from 'react';
import { mount } from 'enzyme';
import ContentError from './ContentError';

describe('<ContentError />', () => {
  it('should match snapshot', () => {
    const wrapper = mount(<ContentError />);
    expect(wrapper).toMatchSnapshot();
  });
});
