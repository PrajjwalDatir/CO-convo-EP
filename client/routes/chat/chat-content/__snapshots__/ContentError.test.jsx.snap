// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentError /> should match snapshot 1`] = `
<ContentError>
  <ContentContainer
    direction="flex-row"
  >
    <div
      className="my-8 flex flex-grow bg-white rounded-lg shadow-xl flex-row"
    >
      <div
        className="flex flex-col items-center justify-center"
      >
        <div
          className="ml-8"
        >
          <h1
            className="text-5xl whitespace-normal leading-tight font-bold"
            style={
              Object {
                "fontFamily": "Raleway, sans-serif",
              }
            }
          >
            A server error has occurred.
          </h1>
          <p
            className="text-gray-700 text-xl mt-2 mb-5"
          >
            Please try again later
          </p>
        </div>
        <img
          alt="Man looking at map"
          className="w-3/5"
          src="test-file-stub"
        />
      </div>
    </div>
  </ContentContainer>
</ContentError>
`;
